backend/tools/scrape.py

```
@@ -0,0 +1,358 @@
+            ]
+        }
+    """
+    return fetch_product_reviews_impl(goods_no, review_count)
+
+
+def fetch_product_reviews_impl(goods_no: str,review_count: int = 3) -> Dict[str, Any]:
+    try:
+        response = session.get(f"https://goods.musinsa.com/api2/review/v1/picture-reviews?goodsNo={goods_no}&size={review_count}&page=1", timeout=10,verify=False)
+        response.raise_for_status()
+        
+        data = response.json();
```

Ruff (0.12.2)

Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)